allennlp/service/predictors/predictor.py

```diff
@@ -1,6 +1,6 @@
 # pylint: disable=unused-import
 import warnings
 
-from allennlp.predictors.predictor import Predictor, DemoModel, DEFAULT_PREDICTORS
+from allennlp.predictors.predictor import Predictor, DEFAULT_PREDICTORS
```

Comment:
Leave the DemoModel import here (change it to it's new location) to preserve backwards compatibility

Reply:
Ultimately I'd like to move the demo to a separate repository that depends on AllenNLP by a pip dependency.  How can I leave the import here but mark it as deprecated?

Reply:
copy the source here or something? 🤷‍♂️

Reply:
I added it to `__init__.py` but I don't understand how adding that import would help backwards compatibility here.  Can you explain/give an example?